lib/assert.js

```diff
@@ -931,6 +932,79 @@ assert.doesNotMatch = function doesNotMatch(string, regexp, message) {
   internalMatch(string, regexp, message, doesNotMatch);
 };
 
+const noop = () => {};
+
+const mustCallChecks = [];
+
+var attachExitHandler = 0;
+
+function runCallChecks(exitCode) {
+  const console = require('console');
+
+  const failed = mustCallChecks.filter((context) => {
+    context.messageSegment = `exactly ${context.exact}`;
```

Comment:
This property does not seem to be necessary. The console error below could just access `context.exact` and add the word `exactly`.